sim-administration/simmanager/src/integration-test/kotlin/org/ostelco/simcards/admin/SimAdministrationTest.kt

```diff
@@ -164,6 +162,11 @@ class SimAdministrationTest {
         dao.clearTables()
     }
 
+    private val hssName = "Foo"
```

Comment:
Why move variables down below the methods?  There were at the top of the class before?